main.yml

```diff
@@ -268,6 +268,10 @@ cost-management:
   frontend:
     paths:
       - /cost-management
+    module:
+      appName: cost-management
+      scope: costManagement
+      module: ./RootApp
```

Comment:
Cost management is a mix of first level and second level navigation. You should add `group: cost-management` in here in order to use correct module if user refreshes page with for instance `/beta/cost-management/ocp` without it the UI would break with error from component.